src/views/Timesheets/components/ShiftItem/index.jsx

```diff
@@ -421,7 +420,7 @@ export default class ShiftItem extends Component {
     );
     let deleteButton = (
       <IconButton onClick={this.handleDeleteButtonClick} size="large">
-        <DeleteForeverOutlinedIcon style={{ color: gordonColors.secondary.red }} />
+        <DeleteForeverOutlinedIcon className={styles.timeshifts_dButton} />
```

Comment:
```suggestion
        <DeleteForeverOutlinedIcon className={styles.timesheets_dButton} />
```
I think you have a typo here, but this will probably be changed by getting rid of the &s in the CSS class.  It also might be good to have more descriptive classnames than just dbutton and ibutton, maybe delete_button and icon_button (or whatever makes more sense)